.github/workflows/maven.yml

```diff
@@ -3,7 +3,7 @@ name: Java Test Coverage with Maven, Coveralls
 on:
     pull_request:
     push:
-        branches: [ master ]
+        branches: [ master, Memory2-doc-fixes-gh-actions ]
```

Comment:
I assume this is temporary until we merge into master.

Reply:
Good spot, this was for testing and I'll remove it now before merging.